src/util/run_command.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
boost process was written in 2014 :eyes:

Reply:
🧙🔮

Reply:
As you pointed out, this is obviously all new code from 2020+, so I've dropped this credit.